docs/installation.rst

```diff
@@ -142,6 +142,7 @@ following dependencies first:
 * OpenSSL_ >= 1.0.0, plus development headers.
 * libacl_ (which depends on libattr_), both plus development headers.
 * liblz4_, plus development headers.
+* ZeroMQ_ >= 4.0.0, plus development headers.
```

Comment:
I expected it would still be required to build pyzmq. There seems to be a debian package for pyzmq, which depends on libzmq. Not sure what to make of that, for now I'd say it's sufficient to note that in one form or another this is required.